src/main/java/baritone/utils/ExampleBaritoneControl.java

```diff
@@ -299,7 +299,7 @@ public boolean isInGoal(BlockPos pos) {
             logDirect("Started mining blocks of type " + Arrays.toString(blockTypes));
             return true;
         }
-        if (msg.startsWith("thisway")) {
+        if (msg.startsWith("thisway") || msg.equals("forward")) {
```

Comment:
I'd be inclined to start using regexes if you want to support multiple aliases:

```suggestion
        if (msg.matches("^(thisway|forward).*")) {
```